core/search/levenshtein_filter.hpp

```diff
@@ -61,16 +61,21 @@ struct IRESEARCH_API by_edit_distance_filter_options {
   //////////////////////////////////////////////////////////////////////////////
   bool with_transpositions{false};
 
+  //////////////////////////////////////////////////////////////////////////////
+  /// @brief match this number of characters from the beginning of the target regardless of edit distance
+  //////////////////////////////////////////////////////////////////////////////
+  bstring prefix;
```

Comment:
the good practice is to sort structure member in descending order of their size, i.e. you have to put `prefix` right after `target`